src/MainWindow.vala

```diff
@@ -132,7 +120,24 @@ public class AppCenter.MainWindow : Gtk.ApplicationWindow {
     }
 
     construct {
+        settings = new GLib.Settings ("io.elementary.appcenter.settings");
+
+        int window_x, window_y;
+        var rect = Gtk.Allocation ();
+
+        settings.get ("window-position", "(ii)", out window_x, out window_y);
+        settings.get ("window-size", "(ii)", out rect.width, out rect.height);
+
+        if (window_x != -1 ||  window_y != -1) {
+            move (window_x, window_y);
```

Comment:
Not sure this will work with all window managers, but as long as it works with gala (which it does at the moment) I guess thats OK.